src/algorithms/clustering/k_means.py

```diff
@@ -81,15 +82,16 @@ def mean(self, group: pd.DataFrame) -> Tuple:
             if self.is_numeric[i]:
                 result.append(column.mean())
             else:
-                counter = {}
-                most_frequent = None
-                count = 0
-                for element in column:
-                    counter[element] = counter.get(element, 0) + 1
-                    if counter[element] > count:
-                        count = counter[element]
-                        most_frequent = element
-                result.append(most_frequent)
+                result.append("NULL")
```

Comment:
will changing "NULL" string to _None_ break something down the line? if not then that would probably be better, but I assume that this part is temporary either way, so it doesn't matter much